wolfssl/openssl/evp.h

```diff
@@ -372,6 +376,39 @@ struct WOLFSSL_EVP_PKEY_CTX {
     int nbits;
 };
 
+#if defined(WOLFSSL_BASE64_ENCODE) || defined(WOLFSSL_BASE64_DECODE)
+
+#define   BASE64_ENCODE_BLOCK_SIZE  48
+#define   BASE64_ENCODED_BLOCK_SIZE 64
```

Comment:
BASE64_ENCODE_BLOCK_SIZE and BASE64_ENCODED_BLOCK_SIZE seem too similar in naming.  I can see them getting easily confused.  Can we rename one slightly?

Reply:
Changed it to BASE64_ENCODE_RESULT_BLOCK_SIZE.